openmetadata-spec/src/main/resources/json/schema/metadataIngestion/objectstore/containerMetadataConfig.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$id": "https://open-metadata.org/schema/metadataIngestion/containerMetadataConfig.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ObjectStoreContainerConfig",
+  "description": "Definition of the properties contained by an object store container template config file",
+  "javaType": "org.openmetadata.schema.metadataIngestion.objectstore.ContainerMetadataConfig",
+  "definitions": {
+    "metadataEntry": {
+      "description": "Config properties for a container found in a user-supplied metadata config",
+      "javaType": "org.openmetadata.schema.metadataIngestion.objectstore.ContainerMetadataEntry",
+      "type": "object",
+      "properties": {
+        "dataPath": {
+          "title": "Data path",
+          "description": "The path where the data resides in the container, excluding the bucket name",
+          "type": "string"
+        },
+        "isStructured": {
+          "title": "Is Structured",
+          "description": "Whether the container defines a supported structure like avro, parquet, csv.",
+          "type": "boolean",
+          "default": false
+        },
+        "structureFormat": {
+          "title": "Schema format",
```

Comment:
if structureFormat is null, then isStructured is false?

Reply:
nice catch. This one we can remove